configure.in

```diff
@@ -8447,6 +8499,14 @@ else
 fi
 SUBDIRS="$SUBDIRS tests"
 
+dnl Check for catch (C++ Automated Test Cases in Headers) availability.
```

Comment:
This is not required for building the library, strictly speaking. Perhaps we should add `--disable-tests` to skip this (and also avoid adding `tests` to `SUBDIRS` above then, of course)?

Reply:
I have to look into it. May need a little time

Reply:
No problem, I can do it myself when merging this PR.

Reply:
Option added. Hopefully position of `--disable-tests` is correct and readable.